### PR TITLE
Cache Sponsor Banner

### PR DESCRIPTION
**Pull request by KarateBrot**
Caches the sponsor banner for the current browser session.

### CODE REVIEW

File: src/components/Sponsors/Sponsors.tsx [hunk 1]
```diff
@@ -3,12 +3,15 @@ import { useColorMode } from '@docusaurus/theme-common';
 import { sanitize } from 'dompurify';
 
 export default function Sponsors(): React.JSX.Element {
-  const [data, setData] = useState(null);
   const { colorMode } = useColorMode();
+  const [data, setData] = useState<string | null>(localStorage.getItem(colorMode));
```
Comment (haslinghuis) [inline src/components/Sponsors/Sponsors.tsx:7]:
```suggestion
  const [data, setData] = useState <string | null> (localStorage.getItem(colorMode));
```

**Review by haslinghuis — APPROVED** [review]
typescript != javascript :)

File: src/components/Sponsors/Sponsors.tsx (hunk 1 above)
Comment (haslinghuis) [reply src/components/Sponsors/Sponsors.tsx:7]: image

Comment (KarateBrot) [reply src/components/Sponsors/Sponsors.tsx:7]: We only have a javascript label. Does it even matter or do we need to introduce a typescript label?

Comment (haslinghuis) [reply src/components/Sponsors/Sponsors.tsx:7]: Added new label.

**Comment by KarateBrot** [comment]
PR is no longer needed as we are now cycling the logos

**Pull request closed without merging**